src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SecurityFactoryInterface.php

```diff
@@ -31,7 +46,18 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
      */
     public function getPosition();
 
+    /**
+     * Defines the configuration key used to reference the provider
+     * in the firewall configuration.
+     * 
+     * @return string
```

Comment:
Can you verify if it's really just the key or if it's the context? The context is the option you can pass along to a firewall in case they share they share context but have a different name. By default the context (if not provided) is the firewall name.